views/scss/inc/_delivery-form.scss

```diff
@@ -0,0 +1,34 @@
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU General Public License
+/// as published by the Free Software Foundation; under version 2
+/// of the License (non-upgradable).
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/// GNU General Public License for more details.
+///
+/// You should have received a copy of the GNU General Public License
+/// along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+/// Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+///
+
+@import "inc/bootstrap";
+
+.delivery-headings {
+    h3 {
+        margin-bottom: 5px;
+    }
+    h2 {
+        margin-bottom: 5px;
+    }
+}
+
+.data-container-wrapper {
```

Comment:
this class is used in other places, shouldn't this be fixed everywhere ?

Reply:
I will address this in a separate PR